docs/source/visualization/interval_color_mapping.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>do you need to add <code>branca</code> to this too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I didn't need to install it via pip, I think it must come with folium</p>

Reply:
Looks like it does : https://github.com/python-visualization/folium/blob/main/requirements.txt.
Could always add it since we have it as a direct dependency in the notebook ?

Reply:
🤔 Is there a standard for whether to explicitly install a package if you have a direct dependency or if you can import it without a direct install when it comes with another package? I'm guessing that it is best practice to have an explicit install if you have a direct dependency.

Reply:
Not sure, just my intuition. It would limit the likelihood of our code failing if `folium` drops `branca` (I don't know anything about these libraries, but I guess that's unlikely 😄).